code/parse/sexp.cpp

```diff
@@ -26281,6 +26283,9 @@ int query_operator_argument_type(int op, int argnum)
 				return OPF_STRING;
 			} else if (argnum == 2) {
 				return OPF_VARIABLE_NAME;
+			} else {
+				// This shouldn't happen
```

Comment:
If it shouldn't happen, consider throwing an `Assertion(false)` in there.

Reply:
I'm pretty sure that that code should never be called with an invalid `argnum` value but if that happens then `OPF_NONE` is the correct return value (at least I think it's correct). I added the comment to signify that the `else` branch should never be called but if it is then it still returns a valid value.